<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeaWindow</class>
 <widget class="QMainWindow" name="LeaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1000</width>
      <height>650</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;华文仿宋&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>最近100条请假记录</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>971</width>
        <height>551</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;华文仿宋&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>班级</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学院</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>请假时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>请假原因</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>审批时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>审批人</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>570</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>查询界面</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>971</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;华文仿宋&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>班级</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学院</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>请假时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>请假原因</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>审批时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>审批人</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>570</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>501</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox_type">
         <item>
          <property name="text">
           <string>学院名称</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>班级</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>姓名</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>学号</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>审批人</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_type"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>22</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox_time">
         <property name="text">
          <string>时间范围</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>至</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_search">
         <property name="text">
          <string>查找</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
